src/passes/shadowstack.ts

```diff
@@ -477,7 +478,9 @@ export class ShadowStackPass extends Pass {
     let moduleRef = this.module.ref;
     _BinaryenRemoveFunction(moduleRef, name);
     let cArr = allocPtrArray(vars);
-    _BinaryenAddFunction(moduleRef, name, params, results, cArr, vars.length, body);
+    let newFuncRef = _BinaryenAddFunction(moduleRef, name, params, results, cArr, vars.length, body);
+    let func = Function.searchFunctionByRef(this.compiler, newFuncRef);
+    if (func) func.addDebugInfo(this.module, newFuncRef);
```

Comment:
```suggestion
    if (this.options.sourceMap || this.options.debugInfo) {
      let func = Function.searchFunctionByRef(this.compiler, newFuncRef);
      if (func) func.addDebugInfo(this.module, newFuncRef);
    }
```
For skipping unnecessary search routine when we don't actually need this.